impl/src/main/java/org/ehcache/internal/persistence/DefaultLocalPersistenceService.java

```diff
@@ -166,6 +166,15 @@ public void destroyPersistenceContext(Object identifier) throws CachePersistence
     }
   }
 
+  @Override
+  public void destroyAllPersistenceContext() {
+    if(recursiveDeleteDirectoryContent(rootDirectory)){
+      LOGGER.info("Destroying all file based persistence context");
```

Comment:
The log should rather say "Destroyed all file based persistence context" instead of "Destroying..." as the logging happens after deletion was done.